setup.py

```diff
@@ -68,11 +68,13 @@ def get_version(rel_path):
         exclude=["contrib", "docs", "tests*", "tasks"],
     ),
     package_data={
+        "pip": ["py.typed"],
         "pip._vendor": ["vendor.txt"],
         "pip._vendor.certifi": ["*.pem"],
         "pip._vendor.requests": ["*.pem"],
         "pip._vendor.distlib._backport": ["sysconfig.cfg"],
         "pip._vendor.distlib": ["t32.exe", "t64.exe", "w32.exe", "w64.exe"],
+        "pip._vendor.packaging": ["py.typed"],
```

Comment:
This shouldn't be necessary -- the py.typed file is meant to be "applies to everything in the namespace" and this is covered by the top level file.